app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -11,10 +11,7 @@ import android.preference.PreferenceManager
 import android.support.v4.view.MenuItemCompat
 import android.support.v7.app.AppCompatActivity
 import android.util.Log
-import android.view.Menu
-import android.view.MenuItem
-import android.view.MotionEvent
-import android.view.View
+import android.view.*
```

Comment:
Let's avoid using * imports since they can be ambiguous. I think IntelliJ/Studio uses * imports by default if there are 5 or more classes from the same package but you can change this in the IDE settings.